web/client/plugins/manager/users/UserDialog.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const {bindActionCreators} = require('redux');
+const {connect} = require('react-redux');
+const {editUser, changeUserMetadata, saveUser} = require('../../../actions/users');
+
+
+const mapStateToProps = (state) => {
```

Comment:
They are only not null checks, how it is less complex and more readable than this?